ci/owncloud/upload_to_owncloud.py

```diff
@@ -0,0 +1,153 @@
+import argparse
+from os import path
+from getpass import getuser
+import netrc
+from webdav3.client import Client
+from uuid import uuid4
+import json
+from opts import debug, message, opts
+
+default_hostname = "https://ftp.essensium.com/owncloud/remote.php/dav/files/"
+
+
+def get_credentials(options_path=None, force_netrc=True):
+    """
+    get_credentials(options_path=None, force_netrc=True)
+
+    Gets the user's default credentials from either an external options file or ~/.netrc
```

Comment:
Don't repeat the function signature, and put the summary line on the same line as the `"""`.
```suggestion
    """Gets the user's default credentials from either an external options file or ~/.netrc
```